### ISSUE TITLE
[Minor][1.16.1] Tool Tooltip doesn't display Hoe as most effective tool

### ISSUE THREAD

**Issue by 04Slash** [McJty/TheOneProbe#394]
Forge/Mod(s):
forge-32.0.57
jei-1.16.1-7.0.0.6.jar

In the 1.16 update, the Hoe has become more effective at mining certain blocks. When looking at these blocks it shows "No Tool" as the most effective tool. 

- sponges,
- wet sponges,
- dried kelp blocks,
- nether wart blocks,
- targets,
- shroomlights,
- leaves,
- hay bales
The above list was grabbed from [the minecraft wiki changelog for 1.16](https://minecraft.gamepedia.com/Java_Edition_1.16).

Thank you for all the time and effort you put into modding minecraft. <3

P.S. The game is literally unplayable because of this. ):<

**Comment by McJty** [comment]
This is actually a bug in Forge:

https://github.com/MinecraftForge/MinecraftForge/blob/1.16.x/src/main/java/net/minecraftforge/common/ForgeHooks.java#L208-L228

**Comment by 04Slash** [comment]
Hey, just looking for some clarification here. I looked through the (roughly) last week of Forge's issues (opened and closed) and didn't really see anything. I found [this](https://github.com/MinecraftForge/MinecraftForge/pull/6872) which adds the hoe as a tooltype, but I assume there is more that needs to be done to fix this?
I have updated to the latest version of forge, which at the time of writing this is 32.0.61 and the issue is still present. I assume I should report this to forge, correct?
